src/components/SimpleList/components/Body.js

```diff
@@ -36,9 +36,10 @@ const Body = ({
                 {headings.map((col: Object, ck: number): React$Element<*> => (
                   <SimpleItemComponent
                     key={`${ck}`}
+                    rowIndex={vk}
                     row={v}
                     text={v[col.name]}
-                    flex={`${(1 / headings.length) * 100}%`}
+                    flex={`${col.flex || (1 / headings.length) * 100}%`}
```

Comment:
adding 'flex' property support here allows the generic SimpleItem to use custom 'width' values as opposed to using homebrew SimpleItems